hubspot/packages/frontend/src/app.d.ts

```diff
@@ -0,0 +1,12 @@
+// See https://kit.svelte.dev/docs/types#app
+// for information about these interfaces
+declare global {
+	namespace App {
+		// interface Error {}
```

Comment:
@srmagura just curious, is this a sample code from create-app template?

Reply:
Yes, it is.